spec/graphql/execution_error_spec.rb

```diff
@@ -296,18 +296,6 @@
   describe "more than one ExecutionError" do
     let(:query_string) { %|{ multipleErrorsOnNonNullableField} |}
     it "the errors are inserted into the errors key and the data is nil even for a NonNullable field " do
-
```

Comment:
This comment makes sense to me because I submitted the PR that introduced this test. The comment is correct (and thank you sincerely for pointing this out).

I've taken the liberty to remove the comment because I see that you have already corrected the spec (and the gem to actually return the expected result below) and I think that at this point the comment is no longer necessary.